.github/workflows/sync_kubeflow_manifests.yaml

```diff
@@ -0,0 +1,27 @@
+name: Sync odh-manifests with notebook controller manifests from kubeflow
+
+on:
+  workflow_dispatch
+
+jobs:
+  kubeflow-sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
+      - name: Gather files from kubeflow and replace manifests 
+        shell: bash
+        run: |
+          cd ${{ github.workspace }}/odh-notebook-controller
+          svn export --force https://github.com/opendatahub-io/kubeflow/branches/v1.6-branch/components/notebook-controller/config ./kf-notebook-controller
+          svn export --force https://github.com/opendatahub-io/kubeflow/branches/v1.6-branch/components/odh-notebook-controller/config ./odh-notebook-controller
```

Comment:
if the goal is to get the config folder from upstream kubeflow to downstream, it is way to complicated to use "svn" and i do not believe by default svn is installed in ubuntu image.